scripts/gtk-bundle-from-msys2.sh

```diff
@@ -111,17 +133,18 @@ ${gtk}
 
 _getpkg() {
 	if [ "$use_cache" = "yes" ]; then
-		package_info=`pacman -Qi mingw-w64-$ABI-$1`
-		package_version=`echo "$package_info" | grep "^Version " | cut -d':' -f 2 | tr -d '[[:space:]]'`
+		package_info=$(pacman -Qi mingw-w64-$ABI-$1)
+		package_version=$(echo "$package_info" | grep "^Version " | cut -d':' -f 2 | tr -d '[[:space:]]')
 		ls $cachedir/mingw-w64-${ABI}-${1}-${package_version}-* | sort -V | tail -n 1
 	else
-		pacman -Sp mingw-w64-${ABI}-${1}
+		# -dd to ignore dependencies as we listed them already above in $packages
```

Comment:
curious about why do we do that manually now?

Reply:
Even though I can't reproduce it right now, I had `pacman -Sp mingw-w64-x86_64-glib2` printing also all dependencies of the "glib2" package and so we ended up with `mingw-w64-x86_64-python` and many many more. Basically this is nice but the Mingw packages have way more dependencies than we need and to keep the resulting bundle as small as possible ignore the package dependencies and define them manually.

Reply:
OK.  I guess so long as everything runs smoothly it's probably OK anyway indeed